src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/Wasm/WasmBuildIntegrationTest.cs

```diff
@@ -0,0 +1,242 @@
+            Assert.Contains("dotnet.wasm", runtime);
+
+            var assemblies = bootJsonData.resources.assembly.Keys;
+            Assert.Contains("blazorwasm.dll", assemblies);
+            Assert.Contains("RazorClassLibrary.dll", assemblies);
+            Assert.Contains("System.Text.Json.dll", assemblies);
+
+            var pdb = bootJsonData.resources.pdb.Keys;
+            Assert.Contains("blazorwasm.pdb", pdb);
+            Assert.Contains("RazorClassLibrary.pdb", pdb);
+            Assert.Null(bootJsonData.resources.satelliteResources);
+        }
+
+        [Fact(Skip = "https://github.com/dotnet/aspnetcore/issues/22975")]
+        public async Task Build_WithBlazorEnableTimeZoneSupportDisabled_DoesNotCopyTimeZoneInfo()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("blazorwasm", additionalProjects: new[] { "razorclasslibrary" });
+            project.Configuration = "Release";
+            project.AddProjectFileContent(
+@"
+<PropertyGroup>
+    <BlazorEnableTimeZoneSupport>false</BlazorEnableTimeZoneSupport>
+</PropertyGroup>");
+
+            var result = await MSBuildProcessManager.DotnetMSBuild(project);
+
+            Assert.BuildPassed(result);
+
+            var buildOutputDirectory = project.BuildOutputDirectory;
+
+            var bootJsonPath = Path.Combine(buildOutputDirectory, "wwwroot", "_framework", "blazor.boot.json");
+            var bootJsonData = ReadBootJsonData(result, bootJsonPath);
+
+            var runtime = bootJsonData.resources.runtime.Keys;
+            Assert.Contains("dotnet.wasm", runtime);
+            Assert.DoesNotContain("dotnet.timezones.dat", runtime);
+
+            Assert.FileExists(result, buildOutputDirectory, "wwwroot", "_framework", "wasm", "dotnet.wasm");
+            Assert.FileDoesNotExist(result, buildOutputDirectory, "wwwroot", "_framework", "wasm", "dotnet.timezones.dat");
+        }
+
+        [Fact]
+        public async Task Build_Hosted_Works()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("blazorhosted", additionalProjects: new[] { "blazorwasm", "razorclasslibrary", });
+            var result = await MSBuildProcessManager.DotnetMSBuild(project);
+
+            Assert.BuildPassed(result);
+
+            var buildOutputDirectory = project.BuildOutputDirectory;
+
+            Assert.FileDoesNotExist(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "blazorwasm.dll");
+
+            var staticWebAssets = Assert.FileExists(result, buildOutputDirectory, "blazorhosted.StaticWebAssets.xml");
+            Assert.FileContains(result, staticWebAssets, Path.Combine("net5.0", "wwwroot"));
+            Assert.FileContains(result, staticWebAssets, Path.Combine("razorclasslibrary", "wwwroot"));
+            Assert.FileContains(result, staticWebAssets, Path.Combine("blazorwasm", "wwwroot"));
+        }
+
+        [Fact]
+        [QuarantinedTest]
```

Comment:
Since so much has changed, would it be fair to unquarantine this as part of the update? Or do you think it's likely still to have issues?

Reply:
We have a process in place for removing tests from quarantine. They need to be passing without failures for 30 days, so lets just follow the process to avoid disrupting the build